application/src/main/resources/org/opentripplanner/apis/gtfs/schema.graphqls

```diff
@@ -4227,6 +4227,16 @@ input InputUnpreferred {
   useUnpreferredRoutesPenalty: Int @deprecated(reason : "Use unpreferredCost instead")
 }
 
+"Filter for including only certain intermediate places."
+input IntermediatePlaceFilterInput {
+  """
+  Filter the intermediate places by stop type.
+  
+  An empty list is not permitted.
+  """
+  stopTypes: [StopType!]!
+}
```

Comment:
I'm starting to dislike these filtering things more every day due to how complex they have to be to solve simple use cases :D. Now, what if we add lets say rentalTypes here, if you want to include only certain stop types and no rental types, what do you need to do? I know I made this suggestion but I'm starting to find flaws in it...

Reply:
I feel that there is a change request in this paragraph but I'm struggling to find it. :)

Lets chat about this in the meeting.